infra/modules/azure_github_environment_bootstrap/id_infra_cd_iam.tf

```diff
@@ -6,6 +6,13 @@ resource "azurerm_role_assignment" "infra_cd_subscription_reader" {
   description          = "Allow ${var.repository.name} Infra CD identity to read resources at subscription scope"
 }
 
+resource "azurerm_role_assignment" "infra_cd_subscription_rbac_admin" {
```

Comment:
this widens the actual rights to everyone who's able to run CD, isn't it?

Reply:
Yes, but at least a PR is needed. I don't like it, but having a common subscription between all domain teams I haven't found a better approach. If you have some idea, they're welcome.